scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -24,10 +49,78 @@ save_env
 
 echo "======== Startup environment - FINISHING ========"
 
-# Saving START_TIME and when job finishes, check if runtime is not lower than 20m
-# If it is lower, sleep the difference. Will not sleep if CRAB3_RUNTIME_DEBUG is set.
-START_TIME=$(date +%s)
+echo "======== Auxiliary Functions - STARTING ========"
+# auxiliary functions, needed when the job terminates
+# we can consider making some of these fuctions less CRAB-oriented and moving them to ./submit_env.sh
+
+print_file() {
+    # similar in spirit to CMSRunAnalysis.py:logCMSSW()
+
+    keepAtStart=1000
+    keepAtEnd=3000
+    maxLineLen=3000
+
+    FILEPATH=$1
+    cutLines=$(expr $(echo "[$FILEPATH]" | wc -L ) + $maxLineLen)
```

Comment:
I am adding the number of characters of `[cmsRun-stdout.log.tmp]` to `maxLineLen`, so that `maxLineLen` only applies to the stdout of cmssw, not to `[cmsRun-stdout.log.tmp]` as well. 

but ok, unnecessary complication, removing